cmd/yggctl/actions.go

```diff
@@ -42,6 +56,81 @@ func generateControlMessageAction(c *cli.Context) error {
 	return nil
 }
 
+func messageJournalAction(ctx *cli.Context) error {
+	// Set the worker message truncate length from the user-provided arguments
+	truncateLength := ctx.Int("truncate-message")
+	if truncateLength <= 0 {
+		return cli.Exit(
+			fmt.Errorf(
+				"cannot retrieve message journal: 'truncate-message' must be a positive number.",
+			),
+			1,
+		)
+	}
+	// Get the user provided 'worker'/'message-id'/'from'/'to' arguments to filter journal entries if provided.
+	selectedPersistent := ctx.Bool("persistent")
+	selectedWorker := ctx.String("worker")
+	selectedMessageID := ctx.String("message-id")
+	selectedFrom := ctx.String("from")
+	selectedTo := ctx.String("to")
+
```

Comment:
These variables are only accessed once after getting them from the
`cli.Context`. Why not just get them from the context when you need them?